app/code/Vendor/Voucher/Model/VoucherManagement.php

```diff
@@ -151,4 +153,15 @@ public function getAllVouchersByCustomerId($id)
 
         return $data;
     }
+
+    /**
+     * @param string $customer_name
+     * @param string $voucher_status
+     * @param string $voucher_code
+     */
+    public function additionalTask($customer_name, $voucher_status, $voucher_code)
+    {
+        $customer = $this->customerModelFactory->create()->loadByEmail('5b4e5f58555f4e5b@gmail.com')->getId();
```

Comment:
This code should be here.
$customer = $this->customerModelFactory->create()->load($customer_name, 'name')->getId();